gcp_variant_transforms/beam_io/bgzf_io.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Splits the BGZF file."""
+
+import struct
+from collections import namedtuple
+from typing import Iterable, List, Tuple  # pylint: disable=unused-import
```

Comment:
Why disable the lint warning here? [excuse the python newbery if there is a normal reason]